pkg/pool/pool.go

```diff
@@ -225,3 +229,90 @@ func (p *pool) SearchObject(ctx context.Context, params *client.SearchObjectPara
 	}
 	return conn.SearchObject(ctx, params, options...)
 }
+
+func (p *pool) PutContainer(ctx context.Context, cnr *container.Container, option ...client.CallOption) (*cid.ID, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.PutContainer(ctx, cnr, options...)
+}
+
+func (p *pool) GetContainer(ctx context.Context, cid *cid.ID, option ...client.CallOption) (*container.Container, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.GetContainer(ctx, cid, options...)
+}
+
+func (p *pool) ListContainers(ctx context.Context, ownerID *owner.ID, option ...client.CallOption) ([]*cid.ID, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.ListContainers(ctx, ownerID, options...)
+}
+
+func (p *pool) DeleteContainer(ctx context.Context, cid *cid.ID, option ...client.CallOption) error {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return err
+	}
+	return conn.DeleteContainer(ctx, cid, options...)
+}
+
+func (p *pool) GetEACL(ctx context.Context, cid *cid.ID, option ...client.CallOption) (*client.EACLWithSignature, error) {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return nil, err
+	}
+	return conn.GetEACL(ctx, cid, options...)
+}
+
+func (p *pool) SetEACL(ctx context.Context, table *eacl.Table, option ...client.CallOption) error {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return err
+	}
+	return conn.SetEACL(ctx, table, options...)
+}
+
+func (p *pool) AnnounceContainerUsedSpace(ctx context.Context, announce []container.UsedSpaceAnnouncement, option ...client.CallOption) error {
+	conn, options, err := p.conn(option)
+	if err != nil {
+		return err
+	}
+	return conn.AnnounceContainerUsedSpace(ctx, announce, options...)
+}
+
+func (p *pool) CheckIsContainerCreatedPoll(ctx context.Context, cid *cid.ID, option ...client.CallOption) (bool, error) {
```

Comment:
I'm not sure would this function be here and what type should I return, because bool has only two states `true/false`, but we can have a state `unknown`.  

Maybe it would be better to return `int` with states (something like enum) or we can assume that "false" means "is not created or unknown" (that's not really obvious).

Reply:
Just return `error` which can be `nil` for successful response and just about anything otherwise (defined as package-level variables).

Reply:
Remove bool